source/digits_hits/include/GateGlobalActor.hh

```diff
@@ -235,6 +235,30 @@ public:
 	 * */
 	void SetEnableProcessName();
 
+	/** Set enable saving parent ID
+	 * */
+	void SetEnableParentID();
+
+	/** Set enable interaction time. This time is equal difference between local time of pre and post step of track. 
+	    Please remember that definition of local time is: time since the track was created.
+	 * */
+	void SetEnableInteractionTime();
+
+	/** Set enable local time.
+	    Please remember that definition of local time is: time since the track was created.
+	 * */
+	void SetEnableLocalTime();
+
+	/** Set enable global time.
+	    Please remember that definition of global time is: time since the event was created.
```

Comment:
so what is the difference between this and 248?

Reply:
My mistake - interaction time is local time difference between post and pre step. Description changed.